### PR TITLE
[incubator/fluentd-cloudwatch] Update fluentd-cloudwatch tag

### PR DESCRIPTION
**Pull request by swibrow**
@kenden 
What this PR does / why we need it:
This affects the incubator/fluentd-cloudwatch chart.

Fix ```standard_init_linux.go:178: exec user process caused "no such file or directory"``` issue
with latest fluentd-cloudwatch version v0.12.43-cloudwatch

### CODE REVIEW

**Comment by swibrow** [comment]
this is actually a breaking change

[https://github.com/fluent/fluentd-kubernetes-daemonset/issues/172](https://github.com/fluent/fluentd-kubernetes-daemonset/issues/172)

File: incubator/fluentd-cloudwatch/values.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 image:
   repository: fluent/fluentd-kubernetes-daemonset
-  tag: v0.12.33-cloudwatch
+  tag: v0.12.43-cloudwatch
```
Comment (davidkarlsen) [inline incubator/fluentd-cloudwatch/values.yaml:3]: can you update the README to match? TIA

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by kenden** [comment]
LGTM

File: incubator/fluentd-cloudwatch/values.yaml (hunk 1 above)
Comment (swibrow) [reply incubator/fluentd-cloudwatch/values.yaml:3]: @davidkarlsen

**Pull request merged**